requirements.txt

```diff
@@ -1,16 +1,16 @@
 django>=1.11.0,<1.12.0
 elasticsearch==5.3.0
 enum34==1.1.6
+git+https://github.com/flywheel-io/gears.git@v0.1.4#egg=gears
+ipython
 jsonschema==2.6.0
-Markdown==2.6.5
+markdown==2.6.5
```

Comment:
The reason for the capitalization is to match the output of `pip freeze`, so that it can be copied verbatim.